scout/constants/case_tags.py

```diff
@@ -33,6 +33,13 @@
     },
 }
 
+CUSTOM_CASE_REPORT_KEY_NAMES = dict(
```

Comment:
For me it is a bit hard to understand and follow what the field of this variable are. What is it, it's very nested in the end